src/Cli/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -92,7 +92,7 @@ private static int ForwardToVSTestConsole(ParseResult parseResult, string[] args
             // one more time, there is no extra hop via msbuild
             convertedArgs.AddRange(settings);
 
-            if (FeatureFlag.Default.IsEnabled(FeatureFlag.ARTIFACTS_POSTPROCESSING))
+            if (!FeatureFlag.Instance.IsDisabled(FeatureFlag.DISABLE_ARTIFACTS_POSTPROCESSING))
```

Comment:
I missed this in the other review. I would rename IsDisabled to IsSet, or IsTrue, that way it is not confusing whether or not it means DISABLE_ARTIFACTS_POSTPROCESSING is Disabled = isEnabled. Or DISABLE_ARTIFACTS_POSTPROCESSING isDisabled = isEnabled.

```suggestion
            if (!FeatureFlag.Instance.IsSet(FeatureFlag.DISABLE_ARTIFACTS_POSTPROCESSING))
```